src/applicants/applicantsRoutes.js

```diff
@@ -9,5 +9,6 @@ router.get("/all/:page", isAuth, applicantsController.getAllApplicants);
 router.get("/:id", isAuth, applicantsController.getApplicant);
 router.delete("/", isAuth, applicantsController.deleteApplicant);
 router.post("/", isAuth, applicantsController.restoreApplicant);
+router.post("/status", isAuth, applicantsController.changeApplicantStatus);
```

Comment:
- Maybe you should add a validation here, that `statusId` and `personId` are required? Backend should always validate these things because the frontend might send by mistake undefined or null.

Reply:
Added validation